src/types/jobDescription.ts

```diff
@@ -1,62 +1,64 @@
-    outputQueue: string
-    outputQueuePriority: number
-    spooledFileAction: string
-    printerDevice: string
-    printText: string
-    jobMessageQueueMaximumSize: number
-    jobMessageQueueFullAction: string
-    syntaxCheckSeverity: string
-    jobEndSeverity: number
-    joblogOutput: string
-    inquiryMessageReply: string
-    messageLoggingLevel: number
-    messageLoggingSeverity: number
-    messageLoggingText: string
-    logClProgramCommands: string
-    deviceRecoveryAction: string
-    timeSliceEndPool: string
-    allowMultipleThreads: string
-    workloadGroup: string
-    aspgrp: string
-    ddmConversation: string
+  jobDescriptionLibrary: string
+  jobDescription: string
+  authorizationName: string
+  jobDate: string
+  accountingCode: string
+  routingData: string
+  requestData: string
+  libraryListCount: number
+  libraryList: string
+  jobSwitches: string
+  textDescription: string
+  jobQueueLibrary: string
+  jobQueue: string
+  jobQueuePriority: number
+  holdOnJobQueue: string
+  outputQueueLibrary: string
+  outputQueue: string
+  outputQueuePriority: number
+  spooledFileAction: string
+  printerDevice: string
+  printText: string
+  jobMessageQueueMaximumSize: number
+  jobMessageQueueFullAction: string
+  syntaxCheckSeverity: string
+  jobEndSeverity: number
+  joblogOutput: string
+  inquiryMessageReply: string
+  messageLoggingLevel: number
+  messageLoggingSeverity: number
+  messageLoggingText: string
+  logClProgramCommands: string
+  deviceRecoveryAction: string
+  timeSliceEndPool: string
+  allowMultipleThreads: string
+  workloadGroup: string
+  aspgrp: string
+  ddmConversation: string
 }
 
 interface JobDescriptionLibrary {
-    sequence: number
-    library: string
+  sequence: number
+  library: string
 }
 
 export class JobDescription extends Base {
-    private jobDescriptionInfo: JobDescriptionInfo | undefined;
-    private jobDescriptionLibraries: JobDescriptionLibrary[] | undefined;
+  // private jobDescriptionInfo: JobDescriptionInfo | undefined;
+  private jobDescriptionInfo: Record<string, string> = {};
+  private jobDescriptionLibraries: JobDescriptionLibrary[] | undefined;
 
-    async fetch(): Promise<void> {
-        const connection = Code4i.getConnection();
-        const content = Code4i.getContent();
-        if (connection && content) {
-            // Job Description
-            const jobDescriptionInfo = await Code4i.getContent().runSQL([`SELECT JOB_DESCRIPTION_LIBRARY "jobDescriptionLibrary",
+  async fetch(): Promise<void> {
+    const connection = Code4i.getConnection();
+    const content = Code4i.getContent();
+    if (connection && content) {
+      // Job Description
+      const [jobDescriptionInfo] = await Code4i.getContent().runSQL([`SELECT JOB_DESCRIPTION_LIBRARY "jobDescriptionLibrary",
                     JOB_DESCRIPTION "jobDescription",
                     AUTHORIZATION_NAME "authorizationName",
                     ifnull(JOB_DATE, '0001-01-01') jobDate,
```

Comment:
Put `jobDate` between double-quotes, otherwise `jobDescriptionInfo.jobDate` below is always undefined.